hybrisprovider.cpp

```diff
@@ -520,6 +524,54 @@ HybrisProvider::HybrisProvider(QObject *parent)
         m_idleTimer.start(QuitIdleTime, this);
     }
 
+    QString xtraUserAgentFileName;
+
+    QSettings settings(XtraConfigFile, QSettings::IniFormat);
+
+    QString xtraServer;
+
+    for(int i = 0; i < MaxXtraServers; i++) {
+        QString key = QString("xtra/XTRA_SERVER_%1").arg(i);
+
+        xtraServer = settings.value(key, "").toString();
+
+        if(xtraServer != "") {
+            m_xtraServers.enqueue(xtraServer);
+        }
+    }
+
+    m_useForcedXtraInject = settings.value("xtra/XTRA_FORCE_INJECT", "").toBool();
+
+    xtraUserAgentFileName = settings.value("xtra/XTRA_USERAGENT_FILE", "").toString();
+
+    if (xtraUserAgentFileName != "") {
+        QFile xtraUserAgentFile(xtraUserAgentFileName);
+
+        if (xtraUserAgentFile.open(QIODevice::ReadOnly)) {
+            m_xtraUserAgent = xtraUserAgentFile.readLine();
+        }
+    }
+
+    if (m_xtraServers.isEmpty()) {
+        QFile gpsConf(QStringLiteral("/system/etc/gps.conf"));
+        if (!gpsConf.open(QIODevice::ReadOnly))
```

Comment:
Sailfish OS style guide recommends braces around even single-line if blocks.  See https://sailfishos.org/wiki/Coding_Conventions

Reply:
Ah, I see this is just code movement, not modification.  Never mind.

Reply:
i can change those as well if preffered